storage/engine/mvcc.go

```diff
@@ -47,11 +47,7 @@ const (
 //  - Key count (count of all keys, including keys with deleted tombstones)
 //  - Value count (all versions, including deleted tombstones)
 //  - Intents (provisional values written during txns)
-type MVCCStats struct {
-	LiveBytes, KeyBytes, ValBytes, IntentBytes int64
-	LiveCount, KeyCount, ValCount, IntentCount int64
-	IntentAge, GCBytesAge, LastUpdateNanos     int64
-}
+type MVCCStats proto.MVCCStats
```

Comment:
Does it need to be a receiver, though? Unless you need to implement an interface you should be able to convert these to functions with a `*proto.MVCCStats` as the first argument.